packages/custom-functions-metadata-plugin/src/customfunctionsplugin.ts

```diff
@@ -2,49 +2,53 @@
 // Licensed under the MIT license.
 
 import { generateCustomFunctionsMetadata, IGenerateResult } from "custom-functions-metadata";
-import * as fs from "fs";
 import * as path from "path";
-import { Compiler, sources, WebpackError } from "webpack";
+import { Compiler, sources, WebpackError, NormalModule } from "webpack";
 
 const pluginName = "CustomFunctionsMetadataPlugin";
 
 type Options = {input: string, output: string};
 
+
```

Comment:
minor, but there's an extra line here now that can be removed.

Reply:
Oops, missed that one when deleting the interface